policyengine_us/variables/gov/states/la/tax/credits/la_child_tax_credit_non_refundable.py

```diff
@@ -0,0 +1,32 @@
+from policyengine_us.model_api import *
+
+
+class la_child_tax_credit_non_refundable(Variable):
```

Comment:
this will be a formula that takes the la_child_expense_tax_credit.py amount and tests for the la_child_expense_tax_credit_refundable_eligible (is not)